invoke/runners.py

```diff
@@ -476,7 +490,7 @@ def start(self, command):
             )
 
     def default_encoding(self):
-        return locale.getpreferredencoding(False)
+        return locale.getpreferredencoding()
```

Comment:
I agree that comment will be useful here, I was considering adding it. However, I believe that since Invoke was developed mostly on Linux, this is not a fix for any platform. I have found the first commit that introduced this behaviour: https://github.com/pyinvoke/invoke/commit/2de58e9e64f742df2539de99a1d58b93b96a7296 BTW, it was your commit! Could you shed some light on it?